index.bs

```diff
@@ -2500,7 +2500,8 @@ To <dfn>match an attribution source's filter data against a filter config</dfn>
 1. Let |lookbackWindow| be |filter|'s [=filter config/lookback window=].
 1. If |lookbackWindow| is not null:
     1. If the [=duration from=] |moment| and the |source|'s [=attribution source/source time=] is greater than |lookbackWindow|:
-        1. If |isNegated|, return true. Otherwise, return false. 
```

Comment:
Could you clarify this change? 

IIUC we always return false in this case now, is that correct?

Reply:
Yes, this is correct. 

When non-negated, we want to check that we are inside the window and the other filters match. We can return early (and false) when we are outside the lookback window.

When negated, we want to check that we are outside and the window and other filters dont match. We can return early (and false) when we are inside the lookback window.

Reply:
As discussed I added a note in the spec to make this section clearer.